### PR TITLE
Add tripleo_deploy target to devsetup Makefile plus scripts

### PR DESCRIPTION
**Pull request by marios**
Add tripleo_deploy target to devsetup Makefile plus scripts

As part of [1] sketching out the tripleo source ha deployment.
Adds a new tripleo_deploy makefile target in devsetup which for now
installs the undercloud

https://issues.redhat.com/browse/OSPRH-4411

[1] https://issues.redhat.com/browse/OSPRH-3038

### CODE REVIEW

File: devsetup/scripts/undercloud.sh [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+EDPM_COMPUTE_SUFFIX=${1:-"0"}
+EDPM_COMPUTE_ADDITIONAL_NETWORKS=${3:-'[]'}
+EDPM_COMPUTE_NAME=${EDPM_COMPUTE_NAME:-"edpm-compute-${EDPM_COMPUTE_SUFFIX}"}
+EDPM_COMPUTE_NETWORK=${EDPM_COMPUTE_NETWORK:-default}
+
+IP_ADRESS_SUFFIX=${IP_ADRESS_SUFFIX:-"$((100+${EDPM_COMPUTE_SUFFIX}))"}
+IP=${IP:-"${EDPM_COMPUTE_NETWORK_IP%.*}.${IP_ADRESS_SUFFIX}"}
+OS_NET_CONFIG_IFACE=${OS_NET_CONFIG_IFACE:-"nic1"}
+GATEWAY=${GATEWAY:-"${EDPM_COMPUTE_NETWORK_IP}"}
```
Comment (marios) [inline devsetup/scripts/undercloud.sh:30]: most of these will go away they are pasted from the original standalone.sh

**Review by cescgina — COMMENTED** [review]
lgtm

**Comment by cescgina** [comment]
recheck

**Pull request merged**